HelloWorld_ST_Sensors/user/heartbeat/heartbeat_logic.cpp

```diff
@@ -0,0 +1,491 @@
+#include "heartbeat_logic.hpp"
```

Comment:
Why hearttbeat logic? Improve naming for header files as well

Reply:
Done. Updated in new PR.